SamplesDashboard/SamplesDashboard/ClientApp/src/views/Samples.tsx

```diff
@@ -172,7 +170,7 @@ function renderItemColumn(item: ISampleItem, index: number | undefined, column:
             return <span>{status} </span>;
 
         case 'Language':
-            return <span><Language sampleName={sampleName} /></span>;
+            return <span>{item.language}</span>;
```

Comment:
Lets make this code consistent. Some cases reference item directly while others read out to a separate variable at the top.

Reply:
Alright I'll definitely correct it.